libr/core/cmd_open.c

```diff
@@ -436,16 +456,18 @@ static void cmd_open_bin(RCore *core, const char *input) {
 
 // TODO: discuss the output format
 static void map_list(RIO *io, int mode, RPrint *print, int fd) {
-	SdbListIter *iter;
 	RIOMap *map;
+	ut32 od;
 	if (!io || !io->maps || !print || !print->cb_printf) {
 		return;
 	}
 	if (mode == 'j') {
 		print->cb_printf ("[");
 	}
 	bool first = true;
-	ls_foreach_prev (io->maps, iter, map) {			//this must be prev
+	od = io->maps->ptop - 1;
+	while ((map = r_oids_oget(io->maps, od))) {
```

Comment:
space before (

Reply:
please leave the style comments for a later phase. Right now I think it's more important to check the correctness of the PR, more than the style.

Reply:
The whole PR should br squashed and rebased for proper review, styling, and warnings are important, because we shouldnt merge anything that doesnt conforms to this. i know it is anoying, its just easier to read if the code follows the style. its just 3 comments, nothing more, travis may say about correctness first. if its not green, and time is not worst it matters to be checked later. those are my priorities when reviewing code. without priorities i would die under a huge TODO